server/views/pages/add-steps.njk

```diff
@@ -0,0 +1,146 @@
+{% extends "../partials/layout.njk" %}
+{% from "govuk/components/radios/macro.njk" import govukRadios %}
+{% from "govuk/components/input/macro.njk" import govukInput %}
+{% from "govuk/components/date-input/macro.njk" import govukDateInput %}
+{% from "govuk/components/button/macro.njk" import govukButton %}
+{% from "govuk/components/select/macro.njk" import govukSelect %}
+{% from "govuk/components/details/macro.njk" import govukDetails %}
+{% from "hmpps/components/date-picker/macro.njk" import hmppsDatePicker %}
+{% from "moj/components/side-navigation/macro.njk" import mojSideNavigation %}
+{% from "../components/error-summary/error-summary.njk" import errorSummary %}
+
+{% set locale = interpolate(locale, {
+    applicationName: applicationName,
+    subject: merge(data.popData, {
+        possessiveName: data.popData.givenName | possessive
+    }),
+    goal: data.goal
+}) %}
+
+{% set pageTitle %}
+    {% if errors %}
+        {{ locale.page.errorTitle }}
+    {% else %}
+        {{ locale.page.title }}
+    {% endif %}
+{% endset -%}
+
+{% block content %}
+<a href="#" class="govuk-back-link">Back</a>
+<div class="govuk-grid-row">
+    <div class="govuk-grid-column-full">
+        <span class="govuk-caption-l">{{ locale.mainHeading.caption }}</span>
+        <h1 class="govuk-heading-l">{{ locale.mainHeading.title }}</h1>
+
+        {{ govukDetails({
+            summaryText: locale.detailHeading,
+            text: "...."
+        }) }}
```

Comment:
I think it was a placeholder for content which I forgot to add. Will add this on Monday

Reply:
added summary text